guides/runbooks/orca-zombie-executions.md

```diff
@@ -36,80 +36,19 @@ If you've enabled the zombie check, set an alert on the metric `queue.zombies`,
 
 # Remediation
 
-## Rehydrate the Queue
+You can run this command to cancel a zombie execution via the Orca admin API. 
+This method is preferrable to the `forceCancelExecution` admin command, since it is more obvious for end-users why the execution was canceled.
```

Comment:
Since you've removed `forceCancelExecution` from this doc, is it still worth citing here?

Reply:
Probably not - will update.